retweets.go

```diff
@@ -51,3 +52,89 @@ func retweetsLookup(ctx context.Context, c *client, id string, opt ...*RetweetsL
 
 	return &retweetsLookup, nil
 }
+
+type Body struct {
+	TweetID string `json:"tweet_id"`
+}
+
+func postRetweet(ctx context.Context, c *client, uid string, tid string) (*PostRetweetResponse, error) {
+	if uid == "" {
+		return nil, errors.New("post retweet by uid: uid parameter is required")
+	}
+	postRetweetPath := timeline + "/" + uid + "/retweets"
```

Comment:
Please don't reuse deferent API endpoint.
This API is different from the timeline API.